refined/shared/src/main/scala/monocle/refined/strings.scala

```diff
@@ -1,16 +1,16 @@
 package monocle.refined
 
-import eu.timepit.refined.api.Refined
+import eu.timepit.refined.api.{Validate, Refined}
 import eu.timepit.refined.string.StartsWith
 import monocle._
 
 object strings extends StringsInstances
 
 trait StringsInstances {
 
-  def startsWith(prefix: String): Prism[String, StartsWithString[prefix.type]] = {
+  def startsWith(prefix: String)(implicit v: Validate[String, StartsWith[prefix.type]]): Prism[String, StartsWithString[prefix.type]] = {
```

Comment:
scalac will complain with "could not find implicit value for parameter v: Validate" if we do not specify it as implicit parameter. The reason is that the implicit which provides the `Validate[String, StartsWith[S]]` instance is a `def` which itself requires an implicit `shapeless.Witness[S]`. If you also fetch a `Witness[prefix.type]` in the method definition and make it implicit, you could also fetch `Validate[String, StartsWith[prefix.type]]` in the method without the need to specify as implicit parameter. The advantage of specifying it as implicit parameter is that scalac will fill in all required implicits automatically and that you don't need to know all the dependencies of the `Validate[String, StartsWith[prefix.type]]` instance.